src/config/templates.toml

```diff
@@ -89,7 +89,7 @@ builtin_op_log_comfortable = 'builtin_op_log_compact ++ "\n"'
 'description_placeholder' = 'label("description", "(no description set)")'
 
 # Hook points for users to customize the default templates:
-'format_short_id(id)' = 'id.shortest(12)'
+'format_short_id(id)' = 'id.shortest(8)'
```

Comment:
Fascinating. I just checked on my machine in the jj repo, and didn't realize Git adjusts it.

It does use 12 on linux, even though most commits seem to have a unique prefix of 6 chars (as computed by jj) and, idly scrolling by, I didn't immediately see a commit that takes more than 7 (I'm sure there are some; it might be fun to estimate how many there are).

On my machine, I use 6. I think some programs stop recognizing commit ids as commit ids if they are shorter than ~7-8 chars, but I was considering using 6 by default for change ids.

Reply:
I think we're all OK with 8.